ArraySorted_Not.java

```diff
@@ -0,0 +1,32 @@
+public class checkArray_Sorted_not {
```

Comment:
Change class name to _CheckArraySortedNot_ to follow CamelCase convention.

Reply:
okay.